### DISCUSSION TITLE
Add pm 2.5 info for China please

### DISCUSSION THREAD

**Discussion by maestropss** [FelixdelasPozas/TrayWeather#63] [Ideas]
Could you add PM2.5 indicator (and may be other pollution indicators status) for Chinese cities?

**Comment by FelixdelasPozas** [comment]
Please elaborate a little more because I don't really understand the question. In the pollution tab the values for PM2.5 should appear in the tooltip when moving the mouse to any of the graph points, like in the next image.

![Clipboard01](https://user-images.githubusercontent.com/12167134/138085389-f1a3f4b0-fd46-4757-b102-a23e5c01de83.png)

In the bottom left of the tooltip you can see the PM2.5 value (1.5 micrograms per cubic meter) with the text in dark blue color. The dark blue line in the graph doesn't appear because is probably behind some other value in the bottom of the graph. Maybe is visible zooming in. 

You want the PM2.5 value for the current time shown someplace else? If the weather provides PM2.5 data for you geographic location, it should appear there. Doesn't OpenWeatherMap provide pollution data for Chinese cities? If it doesn't I can't help with that. 

Regards, 
Félix.

**Comment by maestropss** [comment]
Hello. Sorry, I was not clear enough. Pollution indicators are very important for China. We constantly check them before going out. Especially PM2.5. That would good if there is an option for this indicator to be included into the tooltip and/or icon type.

**Comment by FelixdelasPozas** [comment]
I'll look into customizing the tray icon tooltip text, as you're not the first to ask for some info (other than current temperature) added there. 

Regards, 
Félix.

**Comment by FelixdelasPozas** [comment]
Hi @maestropss .

The [latest release (1.13.0)](https://github.com/FelixdelasPozas/TrayWeather/releases/tag/1.13.0) will allow you to customize the information shown in the tray icon tooltip text. You'll be able to add the current PM2.5 and PM10 values to the tray tooltip text.

Regards, 
Félix.

**Comment by maestropss** [comment]
Thank you. Works perfectly. If you could make a portable version, that would be great
